src/sentry/testutils/factories.py

```diff
@@ -535,12 +536,26 @@ def create_artifact_bundle(
     @classmethod
     @exempt_from_silo_limits()
     def create_release_archive(cls, org, release: str, project=None, dist=None):
-        bundle = cls.create_artifact_bundle(org, release, project)
+        bundle = cls.create_artifact_bundle_zip(org, release, project)
         file_ = File.objects.create(name="release-artifacts.zip")
         file_.putfile(ContentFile(bundle))
         release = Release.objects.get(organization__slug=org, version=release)
         return update_artifact_index(release, dist, file_)
 
+    @classmethod
+    @exempt_from_silo_limits()
+    def create_artifact_bundle(cls, org):
+        bundle = cls.create_artifact_bundle_zip(org.slug)
+        file_ = File.objects.create(name="artifact-bundle.zip")
+        file_.putfile(ContentFile(bundle))
+        artifact_bundle = ArtifactBundle.objects.create(
+            organization_id=org.id,
+            bundle_id=uuid4(),
+            file=file_,
+            artifact_count=2,
```

Comment:
do you mind putting it to default kwarg e.g.  `def create_artifact_bundle(cls, org, artifact_count=2)` so you can override it in tests if you wish?